.github/workflows/examples.yml

```diff
@@ -42,14 +42,12 @@ jobs:
     - name: Regular examples
       run: |
         cargo build --target wasm32-wasi --release
-        cd example_js
-        wasmedge --env 'a=1' --env 'b=1' --dir .:. ../target/wasm32-wasi/release/wasmedge_quickjs.wasm hello.js WasmEdge Runtime
+        wasmedge --env 'a=1' --env 'b=1' --dir .:. target/wasm32-wasi/release/wasmedge_quickjs.wasm example_js/hello.js WasmEdge Runtime
     
     - name: Image examples
       run: |
         cargo build --target wasm32-wasi --release
```

Comment:
`cargo build --target wasm32-wasi --release --features=img`

Reply:
I added the `img` flag and it now complains that `image.js` cannot find the `bird.png` file. I think the JS should always be able to find files in the same directory as itself. In fact, the ES6 module demo works file -- it was able to find the module JS files in the same directory.